adal/changelog.txt

```diff
@@ -1,3 +1,17 @@
+Version 4.1.0-preview
+=============
+
+This release includes: 
+
+- Integrated Windows Auth and Username / Password minor changes, such as improved error messages and http call resilience
+
+Version 4.0.1-preview
```

Comment:
Do we have to mention this even if we didn't release it to nuget? Would be something to add to the release process if we need to make note of msal changes here.